model/src/application/interaction/application_command/data/resolved.rs

```diff
@@ -37,8 +37,16 @@ pub struct InteractionChannel {
 
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub struct InteractionMember {
+    /// Member's guild avatar.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub avatar: Option<String>,
     pub joined_at: Timestamp,
     pub nick: Option<String>,
+    /// Whether the user has yet to pass the guild's [Membership Screening]
```

Comment:
we'll need to make sure membership screening is either linked here or the link is removed.

Reply:
I just copied the docs from the Member struct